engine/src/main/java/org/destinationsol/SolApplication.java

```diff
@@ -173,8 +173,14 @@ public void create() {
         nuiManager = new NUIManager(this, context, commonDrawer, options);
     }
 
+    @SuppressWarnings("checkstyle:ParameterAssignment")
```

Comment:
Warnings are normally there to help you. Don't suppress them unless you have a really good reason to.